tsconfig.json

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
[Sand] - I don't think we are using this config anymore right? If so, lets just remove it.

Reply:
The tsconfig is for configuring how typescript will compile it to javascript on build I believe I could be wrong though

Reply:
Cool - then you are good to land this!